pxtblocks/blocklyloader.ts

```diff
@@ -842,8 +987,8 @@ namespace pxt.blocks {
             // In addition to the user's variables, we also want to display the default
             // variable name at the top.  We also don't want this duplicated if the
             // user has created a variable of the same name.
-            goog.array.remove(variableList, Blockly.Msg.VARIABLES_DEFAULT_NAME);
-            variableList.unshift(Blockly.Msg.VARIABLES_DEFAULT_NAME);
+            goog.array.remove(variableList, lf("_default_var"));
```

Comment:
Same as ``_for_do`` and ``_while_do``, please open an issue and we'll fix this in a future commit.